### ISSUE TITLE
[Storybook] Storybook doesn't seem to have accessibility support

### ISSUE THREAD

**Issue by camilasb** [IBM/carbon-addons-iot-react#549]
Steps:
1. Open the IoT Storybook
2. Try to navigate on the left pane using the keyboard

Actual Result: User is not able to do it properly
Expected Result: User should be able to navigate thru storybook properly using the keyboard
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/20360162/64879621-c267d300-d62c-11e9-999e-4553096f5c7a.gif)

Graphite (from Maximo product) team depends on accessibility to be able to put the product in the market, so we need this support.

**Comment by tay1orjones** [comment]
Thanks for noticing this and opening an issue for it!

Storybook is a separate package that is used to aid in documentation for the components exported from this library. We don't directly control the underlying infrastructure beyond the simple [config parameters and addons](https://github.com/IBM/carbon-addons-iot-react/tree/master/.storybook) we have set. Any issues in regards to accessibility or other problems of storybook itself should be directed to [their repository](https://github.com/storybookjs/storybook).

If you're seeing issues with accessibility of the components themselves, please open an issue for those individually and we'll take a look to surface to the Carbon team or address within this library.

**Comment by camilasb** [comment]
Okay, thank you for the clarification!
